models/bamf_mr_brain_tumor/config/default.yml

```diff
@@ -0,0 +1,115 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: Default configuration for Bamf NNUnet Brain tumor segmentation on MR scans (dicom to dicom)
+
+segdb:
+    triplets:
+        T_EDEMA_MABNORMALITY:
+            code: 79654002
+            meaning: Edema
+            scheme_designator: SCT
+        T_NECROSIS_MABNORMALITY:
+            code: 6574001
+            meaning: Necrosis
+            scheme_designator: SCT
+        T_ENHANCING_MABNORMALITY:
+            code: C113842
+            meaning: Enhancing Lesion
+            scheme_designator: NCIt
+    segments:
+        EDEMA:
+            name: Edema
+            category: C_MORPHOLOGICALLY_ABNORMAL_STRUCTURE
+            type: T_EDEMA_MABNORMALITY
+            color: [140, 224, 228]
+        NECROSIS:
+            name: Necrosis
+            category: C_MORPHOLOGICALLY_ABNORMAL_STRUCTURE
+            type: T_EDEMA_MABNORMALITY
```

Comment:
@LennyN95 is this really supposed to be `MABNORMALITY`, or is it a typo?

Reply:
Also, why no just call those `EDEMA`, `NECROSIS` and `ENHANCING_LESION`?

Reply:
@fedorov yes, it's correct as it is defined in the lines above. We use `MABNORMALITY` consistently as a shorter form of `MORPHOLOGICALLY_ABNORMA` ... long ago decisions, not saying perfect.

For your second question, this is what we're doing for the final segmentation. However, the type & category have more verbose id's by segdb's convention starting with T_ and C_, meaning, it can be any unique string, however, when we declare it within the SegDB repo we follow this convention.

Reply:
> We use MABNORMALITY consistently as a shorter form of MORPHOLOGICALLY_ABNORMA ... long ago decisions, not saying perfect.

Hmmm.... Well, good luck explaining how this is easier to understand than DICOM! 😄

Reply:
It's only about reusable ID's, the complexity of DICOM remains :)

Reply:
@jithenece the folowing IDs have been added to SegDB per your definition: `EDEMA`, `NECROSIS` and `ENHANCING_LESION`. 

Side note: for `NECROSIS` I used `T_NECROSIS_MABNORMALITY`.